packages/react-client/example/src/Phone.tsx

```diff
@@ -13,8 +13,6 @@ function Phone() {
         destinationNumber: destination,
         callerName: process.env.REACT_APP_TELNYX_PHONE_NUMBER || '',
         callerNumber: process.env.REACT_APP_TELNYX_PHONE_NUMBER || '',
-        remoteCallerName: '',
-        remoteCallerNumber: '',
```

Comment:
We don't currently document these options, can you confirm that we don't support them @DeividVeloso ?